.github/ISSUE_TEMPLATE/bug.yml

```diff
@@ -17,6 +17,16 @@ body:
       description: >
         Shown in the title bar, logs, and long press on the game icon.  If it's outside a game, put 'None.'
 
+ type: input
+    id: Patched_Iso
+    validations:
+      required: true
+    attributes:
+      label: Game or games this happens in
+      placeholder: No
+      description: >
+        Does the ISO is patched in translation\undub? Please note that patched ISO is not officially supported.
```

Comment:
I'm worried yaml treats \ specially (not a yaml expert.)  Maybe this would be better as "Using a fan translation patch, undub, or modified ISO?  Please note bugs in these are often for the patch authors to fix."

-[Unknown]